docs/transpiling.md

```diff
@@ -31,10 +33,27 @@ If you are using [ts-node](https://github.com/TypeStrong/ts-node):
 
 #### ESM
 
-For ESM projects, you can use `ts-node`'s ESM loader and then `import` your TypeScript files:
+For ESM projects, you can use `ts-node`'s ESM loader and then import your TypeScript files. Doing this will require setting an environment variable. the cleanest way to do this is to include the [cross-env](https://www.npmjs.com/package/cross-env) package with `npm i -D cross-env`
+
+Then change your npm test script invocation in the package.json file to:
+
+```json
+{
+  "scripts: {
```

Comment:
Missing a double quote on the key here

Reply:
Ah.  Sorry.  Also spotted an uncapitalized T at the start of a sentence so I sent the whole thing through a spell/grammar checker to be sure. Should be good now.